processor/batchprocessor/batch_processor_1_22.go

```diff
@@ -0,0 +1,12 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !1.23
```

Comment:
Shouldn't this be `go1.23`? At least on https://stackoverflow.com/a/38439941 that is the format used. Has the format changed?

Reply:
For builds using  go version 1.23 and above , file tagged with `//go:build 1.23` will be used, for go version below 1.23 , file tagged with `//go:build !1.23` will be used.

Reply:
Right, what I meant is that I expected this to be
```suggestion
//go:build !go1.23
```

or at least this is how it used to work for Go in older versions, not `1.23`